test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -192,7 +192,7 @@ public void MulticlassSdcaTest()
         /// <summary>
         /// Multiclass Logistic Regression test with a tree featurizer.
         /// </summary>
-        [Fact]
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
```

Comment:
>[ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline [](start = 8, length = 110)

Let's add an issue to create baselines for those x86 tests, after checking that the difference is ok.

Reply:
https://github.com/dotnet/machinelearning/issues/1215